src/main/rx/cc2500_frsky_x.c

```diff
@@ -276,9 +277,16 @@ void frSkyXSetRcData(uint16_t *rcData, const uint8_t *packet)
         } else {
             j = 0;
         }
+        if (c[i] == 0)
+        {
+            continue;
+        }
         int16_t temp = (((c[i] - 64) << 1) / 3 + 860);
         if ((temp > 800) && (temp < 2200)) {
             rcData[i+j] = temp;
+        } else {
+            DEBUG_SET(DEBUG_RX_FRSKY_SPI, DEBUG_DATA_ERROR_COUNT, ++rangeErrors);
```

Comment:
Having the side-effect inside `DEBUG` is intended?

Reply:
You mean the ++rangeErrors or having a else just for the DEBUG_SET?

Reply:
Increment inside macro.

Reply:
I'll clean it up here.